templates/tfengine/recipes/project.hcl

```diff
@@ -136,6 +136,13 @@ schema = {
             }
           }
         }
+        labels = {
```

Comment:
Follow the existing way of specifying resource labels? e.g. https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/blob/master/templates/tfengine/recipes/resources.hcl#L83